### ISSUE TITLE
Ssh connections refused 

### ISSUE THREAD

**Issue by nobita-gamerz** [tomxi1997/termux-lxc#2]
$ ssh -p 6620 root@localhost
ssh: connect to host localhost port 6620: Connection refused

I'm install with magist and not compile kernel!
Need compile kernel?

Thanks 👍🏻

**Comment by tomxi1997** [comment]
Yes,that need kernel support ,otherwise lxc don't work, you can see those

https://github.com/tomxi1997/lxc-docker-support-for-android

https://github.com/tomxi1997/LXC_KernelSU_Action

**Comment by tomxi1997** [comment]
1
